packages/terra-functional-testing/lib/cli/index.js

```diff
@@ -0,0 +1,3 @@
+const CLI = require('./test-runner-cli');
```

Comment:
I'm curious, why use this file as an import-export pass through? Why not use this file for the CLI logic as well?

Reply:
I think responsibilities of the CLI may continue to grow. I like the consistency of the directories, index files, and specific files behaving in an anticipated way